tasks/setup_wp_database.yml

```diff
@@ -57,6 +58,7 @@
       login_host: "{{ wordpress_mysql_host }}"
       login_user: "{{ wordpress_mysql_user }}"
       login_password: "{{ wordpress_mysql_password }}"
+      login_unix_socket: "{{ mariadb_socket if wordpress_mysql_use_socket else None }}"
```

Comment:
I don't think we should tightly couple two of our roles. Instead, this role should have its own variable and the user can set it to `{{ mariadb_socket }}` if desired.

```suggestion
      login_unix_socket: "{{ wordpress_mysql_socket if wordpress_mysql_socket else None }}"
```

Reply:
This was the previous behavior before #35, but ok I can change it.

Reply:
Problem with this is: It breaks backwards compatibility.
Also: I think the socket path should be defined per host not per wordpress instance

Reply:
Why per host?

Reply:
Because usually there is only one mysql instance on one host, and therefore only one socket on that host.
Configuring it for every instance would be significant configuration overhead.

Reply:
For this we usually have another variable (like `wordpress_default_mysql_socket`) that sets a default for the host, right? Then I don't really care if we support `wp_instance.mysql_socket` or not.